### ISSUE TITLE
Elder Atmosian statue uncraftable?

### ISSUE THREAD

**Issue by iprice** [tgstation/tgstation#74534]
Reporting client version: 514.1589

## Round ID:
203201

## Testmerges:
- [Disables auxtools](https://www.github.com/tgstation/tgstation/pull/74497)
- [Clients can't OOC until they have fully gone through `New()`](https://www.github.com/tgstation/tgstation/pull/74493)
- [The North Star Expeditionary Vessel - A Second Wind](https://www.github.com/tgstation/tgstation/pull/74371)
- [Adds Master.sleep_delta to the mc tab.](https://www.github.com/tgstation/tgstation/pull/74360)
- [Imports and Contraband: Different! Cargo crates without locks! MEAT!](https://www.github.com/tgstation/tgstation/pull/74490)

## Reproduction:

The Elder Atmosian statue no longer seems to be in the crafting menu?  Perhaps since the recent UI changes to it?

Or I'm missing a trick, if so, please let me know!

**Comment by timothymtorres** [comment]
Relevant code:

https://github.com/tgstation/tgstation/blob/2f16144456cad71c5e5ff06bba0e8620dc861cc2/code/modules/art/statues.dm#L244-L253

Did you use metal hydrogen?

**Comment by iprice** [comment]
> Did you use metal hydrogen?

Yup.  And even if I didn't I should still be able to find it in the crafting menu, just not actually craft it?  It's just not there at all.
